san/transform.py

```diff
@@ -11,9 +11,78 @@
 QUERY_PATH_MAP = {
     'eth_top_transactions': ['ethTopTransactions'],
     'eth_spent_over_time': ['ethSpentOverTime'],
-    'token_top_transactions':['tokenTopTransactions']
+    'token_top_transactions': ['tokenTopTransactions'],
+    'get_metric': ['timeseriesData']
 }
 
+V2_METRIC_QUERIES = [
+    'daily_active_addresses',
+    'daily_avg_marketcap_usd',
+    'daily_avg_price_usd',
+    'daily_closing_marketcap_usd',
+    'daily_closing_price_usd',
+    'daily_high_price_usd',
+    'daily_low_price_usd',
+    'daily_opening_price_usd',
```

Comment:
remove all price-related metrics from here

Reply:
I guess without realized price, I will do it